client/my-sites/migrate/section-migrate.jsx

```diff
@@ -72,6 +73,32 @@ class SectionMigrate extends Component {
 		this.props.navigateToSelectedSourceSite( sourceSiteId );
 	};
 
+	getStartTime = () => {
+		if ( ! window.localStorage ) {
+			return;
+		}
+
+		return window.localStorage.getItem( 'siteMigrationStartTime' );
+	};
+
+	setStartTime = () => {
+		if ( ! window.localStorage ) {
+			return;
+		}
+		const localeSlug = getLocaleSlug();
+		const startTime = moment()
+			.locale( localeSlug )
+			.format( 'LT D MMMM' );
```

Comment:
This technique doesn't result in something that's really a localized string. For example, it results in "Migration started 2:50 PM 9 December" for me. This isn't a format that would ever really be seen in the US.

Maybe we should use `LLLL` or `llll` instead.

Reply:
Ah right! The `LLLL` and `llll` formats seem a bit excessively detailed – we could go with `lll`, which is `June 9 2014 9:32 PM` in my locale.